### PR TITLE
CORE 2001 allow to run only parts of setup i

### PR DESCRIPTION
**Pull request by stereomon**
Added new commands to delete/purge queues and to delete exchanges

### CODE REVIEW

**Comment by dereuromark** [comment]
Do you also get
```
FILE: ...t-mq/src/Spryker/Client/RabbitMq/Model/Connection/Connection.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 95 | ERROR | Doc Block param variable `$exchangeQueueName` should be
    |       | `$queueOption`
 96 | ERROR | Doc Block param variable `$queueOption` should be
    |       | `$exchangeQueueName`
```
?

File: src/Spryker/Zed/RabbitMq/Business/Model/Queue/Queue.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Business\Model\Queue;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Client\Queue\Model\Adapter\AdapterInterface;
+
+class Queue implements QueueInterface
```
Comment (ehsanmx) [inline src/Spryker/Zed/RabbitMq/Business/Model/Queue/Queue.php]: Why do you need such class `Queue` and it has only deleting/purging functionality inside, maybe you can change the name to something else?

File: src/Spryker/Zed/RabbitMq/Business/Model/Exchange/Exchange.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Business\Model\Exchange;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Client\Queue\Model\Adapter\AdapterInterface;
+use Spryker\Zed\RabbitMq\Business\Model\Exchange\Filter\ExchangeFilterInterface;
+
+class Exchange implements ExchangeInterface
```
Comment (ehsanmx) [inline src/Spryker/Zed/RabbitMq/Business/Model/Exchange/Exchange.php]: I think it's better to rename `Exchange` and `Queue` classes to something else, related to deletion stuff, they have only deletion and purging?

File: src/Spryker/Zed/RabbitMq/Business/Model/Queue/QueueInterface.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+     * @return bool
+     */
+    public function deleteAllQueues(LoggerInterface $logger);
+
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     *
+     * @return bool
+     */
+    public function purgeAllQueues(LoggerInterface $logger);
```
Comment (ehsanmx) [inline src/Spryker/Zed/RabbitMq/Business/Model/Queue/QueueInterface.php]: Do you think this is good idea to have dependency to logger here?

File: src/Spryker/Zed/RabbitMq/Communication/Console/DeleteAllQueuesConsole.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        $this->setDescription(self::DESCRIPTION);
+
+        parent::configure();
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int|null
```
Comment (ehsanmx) [inline src/Spryker/Zed/RabbitMq/Communication/Console/DeleteAllQueuesConsole.php]: there is no null

File: src/Spryker/Zed/RabbitMq/RabbitMqConfig.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+use Spryker\Zed\RabbitMq\Business\Model\Exchange\ExchangeInfo;
+
+class RabbitMqConfig extends AbstractBundleConfig
+{
+    /**
+     * @return string
+     */
+    public function getApiExchangesUrl()
+    {
+        return sprintf('http://%s:%s/api/exchanges', 'localhost', 15672);
```
Comment (ehsanmx) [inline src/Spryker/Zed/RabbitMq/RabbitMqConfig.php]: I would like to have this in environment config

Comment (dereuromark) [reply src/Spryker/Zed/RabbitMq/RabbitMqConfig.php]: Also, shouldnt we use either https or at least rel // by default?

**Pull request merged**